src/ConfigurationsProvider.tsx

```diff
@@ -32,7 +48,7 @@ export const ConfigurationsContext = createContext(defaultConfiguration);
 export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) => {
   const { globalSettings } = useContext(SettingsContext);
   const { settings, appDesignSystem = {} } = globalSettings;
-  const { sue = {} } = settings || {};
+  const { sue = {}, showResourceFilters = false } = settings || {};
```

Comment:
do we need `showResourceFilters`? each filter has an `active` option, which we need to use. a global setting seems to be unnecessary for me right now

Reply:
Yes, that's right. The configurations in `ConfigurationsProvider` are determined according to `settings.sue`. If an app does not have a `settings.sue` object, then we return the config data by default. That's why I needed a general control for the filter too.

```
...
if (!Object.keys(sue).length || !showResourceFilters) {
      return defaultConfiguration;
    }
...
```